pkg/operator/operator.go

```diff
@@ -305,6 +306,19 @@ func New(config operatorconfig.Config, kubeConfig *rest.Config) (*Operator, erro
 		return nil, fmt.Errorf("failed to create gatewayapi controller: %w", err)
 	}
 
+	// Set up the ingress CA bundle configmap controller
+	if _, err := cabundlecontroller.New(mgr, cabundlecontroller.Config{
+		ServiceCAConfigMapName: operatorcontroller.ServiceCAConfigMapName(),
+		AdminCAConfigMapName:   operatorcontroller.AdminCAConfigMapName(),
+		IngressCAConfigMapName: operatorcontroller.IngressCAConfigMapName(),
+		OperatorNamespacedName: types.NamespacedName{
```

Comment:
Agree. Since operator and default IngressController are in same namespace, named as `OperatorNamespacedName` by tweaking the constant name defined for the namespace